maestro/financial-connections/Testmode-PaymentIntent-TestInstitution-Networking-ReturningUser.yaml

```diff
@@ -1,51 +0,0 @@
-appId: com.stripe.android.financialconnections.example
-tags:
-  - all
-  - edge
-  - testmode-payments
----
-- startRecording: ${'/tmp/test_results/testmode-paymentintent-testinstitution-networking-returning-user-' + new Date().getTime()}
-- clearState
-- openLink: stripeconnectionsexample://playground?flow=PaymentIntent&financial_connections_override_native=native&merchant=networking_testmode&permissions=transactions,payment_method
-- tapOn:
-    id: "Customer email setting"
-- inputText: "email@email.com"
-- hideKeyboard
-- tapOn:
-    id: "connect_accounts"
-# Wait until the consent button is visible
-- extendedWaitUntil:
-    visible:
-      id: "consent_cta"
-    timeout: 30000
-- tapOn:
-    id: "consent_cta"
-# LINK WARMUP PANE
-- extendedWaitUntil:
-    visible:
-      id: "existing_email-button"
-    timeout: 5000
-- assertNotVisible:
-    id: "top-app-bar-back-button"
```

Comment:
We should move this check (and the `assertVisible` one below) over to the other test.

Reply:
good catch!

Reply:
https://github.com/stripe/stripe-android/pull/8050/commits/0e347a9dd6774344d92ebd6162c200bad3450b60